pom.xml

```diff
@@ -72,12 +81,56 @@
       <artifactId>vertx-rx-java</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.termd</groupId>
+      <artifactId>termd-core</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+    </dependency>
 
-    <!-- Not needed at runtime unless you use the JUnit extension -->
+    <!-- -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <groupId>io.vertx</groupId>
```

Comment:
you have a dependency on dropwizard-metrics. Is it mandatory ? How would it work against another implementation of the metrics SPI (hawkular).